client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/EditInteractionEventDialog/EditInteractionEventDialog.tsx

```diff
@@ -20,7 +20,9 @@ const EditInteractionEventDialog : React.FC<Props> = (props: Props) : JSX.Elemen
     const classes = useStyles();
     
     const [interactionEventDialogData, setInteractionEventDialogData] = useState<InteractionEventDialogData>(eventToEdit);
-    const { placeType, placeSubType } = interactionEventDialogData;
+    const [canConfirm, setCanConfirm] = useState<boolean>(false);
```

Comment:
where is this state changes?

Reply:
For now it doesn’t because there are no validations, this is just for now

Reply:
maybe you should export it to a useMemo if it is for future validations